packages/lorcana-engine/src/types/card-types.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * Card Types (Section 6)
+ *
+ * Four card types with distinct behaviors and properties.
+ */
+
+import type { Classification } from "./classifications";
+import type { InkType } from "./ink-types";
+import type { Keyword } from "./keywords";
+
+/** Card Types (Section 6) */
+export const CARD_TYPES = ["character", "action", "item", "location"] as const;
+export type CardType = (typeof CARD_TYPES)[number];
+
+/** Action subtypes */
+export type ActionSubtype = "song" | null;
+
+/**
+ * Ability Definition (placeholder - will be expanded in Spec 7)
+ */
+export interface AbilityDefinition {
+  id: string;
+  name?: string;
+  text: string;
+  type: "triggered" | "activated" | "static";
+}
+
+/**
+ * Lorcana Card Definition (Rule 6.2)
+ *
+ * All card properties including:
+ * - id, name, version, fullName
+ * - inkType (single or dual)
+ * - cost, inkable (inkwell symbol)
+ * - cardType
+ * - Character-specific: strength, willpower, lore, classifications
+ * - Action-specific: actionSubtype (Song)
+ * - Location-specific: moveCost
+ * - keywords, abilities
+ */
+export interface LorcanaCardDefinition {
```

Comment:
Create specific type for Action, Character, Location and Item for stronger type safety. There's a sample in the codebase